app/src/main/java/org/mozilla/focus/widget/Cursor.java

```diff
@@ -50,14 +54,23 @@ public void run() {
     };
 
 
+    private Handler cursorHandler = new Handler();
+    private Runnable cursorTick = new Runnable() {
+        @Override
+        public void run() {
+            animate().alpha(0f).setDuration(CURSOR_ANIMATION_DURATION);
+        }
+    };
+
+
     public Cursor(Context context, AttributeSet attrs) {
         super(context, attrs);
 
         isInit = true;
         // create the Paint and set its color
         paint = new Paint();
         paint.setStyle(Paint.Style.FILL);
-        paint.setAlpha(102);
+        paint.setAlpha(CURSOR_ALPHA);
```

Comment:
nit: -> CURSOR_MAX_ALPHA or similar

Reply:
Since the cursor alpha never changes (It's the entire view that animates out). Does this still make sense to change?

Reply:
Oh, I see – it's fine the way it is. :)